tests/components/smartthings/test_climate.py

```diff
@@ -819,8 +819,8 @@ async def test_updating_humidity(
             Attribute.SUPPORTED_THERMOSTAT_MODES,
             ["coolClean", "dryClean"],
             ATTR_HVAC_MODES,
+            [HVACMode.AUTO],
             [],
-            [HVACMode.COOL, HVACMode.DRY],
         ),
```

Comment:
@joostlek While this PR fixes the issue, the adjustment to this test seems odd and probably reveals a different issue.

I think this virtual thermostat should have been detected as an AC? A capability missing in `AC_CAPABILITIES` maybe?

Reply:
I think its mostly that this was a mistake when writing the tests as coolClean and dryClean aren't thermostat modes. 

The difference with it being a thermostat vs it being an AC is that its sourcing the values from AC related values and not from thermostat ones